packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -1139,6 +1150,7 @@ Individual .babelrc files were generated for you in src/client and src/server
 }
 
 module.exports = function(xclap) {
+  setWebpackDevProfile();
```

Comment:
There are many tasks that don't involve webpack.  You should only set this if the task ends up invoking webpack.  

And you would need the special code in `setWebpackDevProfile` to check what profile to set, because the specific task itself would know and you can set it there.

Reply:
you would *not* need

Reply:
Now, it's set at the specific task that is currently executed. But only if it's required.